src/Replicator.js

```diff
@@ -180,7 +180,8 @@ class Replicator extends EventEmitter {
     this.emit('load.progress', this._id, hash, latest, null, this._buffer.length)
 
     // Return all next pointers
-    return log.values.map(getNext).reduce(flatMap, [])
+    const refsPointers = log.values.map(nextRefsIntersection).reduce(flatMap, [])
+    return refsPointers
```

Comment:
this variable declaration isnt necessary and introduces more changes than just changing the .map callback function